src/Integration.Vsix.UnitTests/CFamily/MessageHandlerTests.cs

```diff
@@ -95,6 +106,7 @@ public void HandleMessage_IssueProcessing_InactiveRulesAreIgnored_ActiveRulesAre
             // Now stream an active rule message
             testSubject.HandleMessage(activeRuleMessage);
 
+            testSubject.AnalysisSucceeded.Should().BeTrue();
```

Comment:
Updated the existing message handler tests: they should now set the `AnalysisSucceeded` property appropriately.